src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/AuthorizeInstancesHelper.cs

```diff
@@ -0,0 +1,289 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
+using Altinn.Common.PEP.Constants;
+using Altinn.Common.PEP.Helpers;
+using Altinn.Common.PEP.Interfaces;
+using Altinn.Platform.Storage.Interface.Models;
+
+namespace Altinn.Platform.Storage.Helpers
+{
+    public class AuthorizeInstancesHelper
```

Comment:
Could we rename to "authorizationHelper" that way it can be extended to include methods for custome authorization of other elements later?